platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r85/ProblemProgressEventCrossVersionTest.groovy

```diff
@@ -54,10 +54,7 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
         then:
         thrown(BuildException)
         def problems = listener.problems.collect { new JsonSlurper().parseText(it.descriptor.json) }
-        problems.size() == 2
-        problems.every {
-            it == [:]
-        }
+        problems.size() == 0
```

Comment:
🤔 I do wonder, why did we lose all the problems here?

Reply:
Probably because of the incompatible changes we do in the protocol. We make sure that old clients don't blow up, but since this is an incubating API, it's less important to make older clients compatible with our latest changes.